docs/en/ingest-management/elastic-agent/advanced-kubernetes-managed-by-fleet.asciidoc

```diff
@@ -0,0 +1,88 @@
+[[advanced-kubernetes-managed-by-fleet]]
+= Advanced {agent} configuration managed by {fleet}
+
+For basic {agent} managed by {fleet} scenarios follow the steps on <<running-on-kubernetes-managed-by-fleet>> page.
+
+On managed {agent} installations could be useful to provide the ability to configure more advanced options like the configuration of providers during the startup. Refer to <<providers>> for more details.
+
+Following steps demonstrate above scenario:
+
+[discrete]
+== Step 1: Download the {agent} manifest
+
+It is advisable to follow the steps of <<install-fleet-managed-elastic-agent>> with Kubernetes Integration installed in your policy and download the {agent} manifest from Kibana UI
+
+image::images/k8kibanaUI.png[{agent} with K8s Package manifest]
```

Comment:
```suggestion
image::images/k8skibanaUI.png[{agent} with K8s Package manifest]
```
Testing locally the docs build failed because it couldn't find this image. This should fix that.